client/src/components/AuthComponent.jsx

```diff
@@ -62,13 +69,30 @@ const AuthComponent = () => {
     const validEmail =
       /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
     if (!validEmail.test(email) || password.length < MIN_PASSWORD_LENGTH) {
-      setShowErrorMessage(true);
+      setShowErrorMessage(
+        'Incorrect email or password. All passwords must be at least 6 characters.'
+      );
     } else if (rememberMe) {
       secureLocalStorage.setItem('email', email);
       secureLocalStorage.setItem('password', password);
       secureLocalStorage.setItem('rememberMe', rememberMe);
     }
   };
+  const googleSignUp = () => {
+    signInWithPopup(auth, provider)
+      .then((result) => {
+        setAuthUser(result.user.email);
+        // add code to store or check user in database
+        secureLocalStorage.setItem('email', result.user.email);
+        setShowErrorMessage('');
```

Comment:
Is this line necessary? If you're navigating to the search page it doesn't matter right?